app/controllers/boxes.py

```diff
@@ -0,0 +1,63 @@
+import json
+
+from flask import Blueprint, jsonify
+
+from app import db
+from app.schemas.boxes import BoxSchema
+from app.schemas.votes import VoteSchema
+from app.models.boxes import BoxModel
+from app.models.options import OptionModel
+from app.models.votes import VoteModel
+from app.helpers import validate_input, token_required
+
+boxes_blueprint = Blueprint("boxes", __name__)
+
+
+@boxes_blueprint.route("/boxes", methods=["GET"])
+@token_required
+def get_all_boxes(user_id):
+    list_boxes = db.session.query(BoxModel).filter(BoxModel.user_id == user_id).all()
+
+    return jsonify(BoxSchema(many=True).dump(list_boxes)), 200
+
+
+@boxes_blueprint.route("/boxes", methods=["POST"])
+@token_required
+@validate_input(schema=BoxSchema)
+def create_new_box(user_id, data):
+    name = data['name']
+    description = data['description']
+
+    # TODO: status of the box (Open/closed)
+
+    new_box = BoxModel(user_id, name, description)
+    db.session.add(new_box)
+    db.session.commit()
+
+    return jsonify(BoxSchema().dump(new_box)), 200
```

Comment:
A status code `201` is more appropriate here.

See [MDN Web Docs](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201)

> The HTTP 201 Created success status response code indicates that the request has succeeded and has led to the creation of a resource. The new resource is effectively created before this response is sent back and the new resource is returned in the body of the message, its location being either the URL of the request, or the content of the Location header.

```suggestion
    return jsonify(BoxSchema().dump(new_box)), 201
```